chain-tx-enclave-next/tdbe/tdbe-common/src/lib.rs

```diff
@@ -10,3 +11,10 @@ pub struct TdbeConfig {
     /// Transaction IDs to fetch from another TDBE server
     pub transaction_ids: Vec<TxId>,
 }
+
+const LIGHT_GENESIS: &str = env!("LIGHT_GENESIS");
```

Comment:
if "light_genesis" is in json, it'll be fairly error-prone (escaping-characters, length limitations on environment variables, etc.) -- `include_str!` may be better

Reply:
done